deployment_script.py

```diff
@@ -5,11 +5,11 @@
 from scripts._0_0_5_remove_old_adm_data import remove_old_adm_records
 from scripts._0_0_6_set_admin_users import add_admin_user_role
 from scripts._0_0_7_update_scenario_index import update_scenario_index
-
```

Comment:
The other scripts have already run on production so we should remove anything 0.0.7 or earlier.